src/main/webapp/app/core/about-us/about-us.component.html

```diff
@@ -66,10 +66,29 @@ <h3 class="font-weight-bold">{{ 'artemisApp.aboutUsOverview.contributors' | arte
         </div>
     }
     <div class="mt-5 ms-2 me-2">
-        <h3 class="font-weight-bold">{{ 'artemisApp.git.title' | artemisTranslate }}</h3>
+        <h3 class="font-weight-bold" jhiTranslate="artemisApp.git.title"></h3>
         <div>
-            {{ 'artemisApp.git.branch' | artemisTranslate }}: {{ gitBranchName }}<br />
-            {{ 'artemisApp.git.commit' | artemisTranslate }}: {{ gitCommitId }}
+            <div class="inline-block fixed-width" jhiTranslate="artemisApp.git.branch"></div>
+            <div class="inline-block">{{ gitBranchName }}</div>
+        </div>
+        <div>
+            <div class="inline-block fixed-width" jhiTranslate="artemisApp.git.commit"></div>
+            <div class="inline-block">{{ gitCommitId }}</div>
+        </div>
+    </div>
+    <div class="mt-5 ms-2 me-2">
+        <h3 class="font-weight-bold" jhiTranslate="artemisApp.universityInfo.title"></h3>
+        <div>
+            <div class="inline-block fixed-width" jhiTranslate="artemisApp.universityInfo.name"></div>
+            <div class="inline-block">{{ universityName }}</div>
+        </div>
+        <div>
+            <div class="inline-block fixed-width" jhiTranslate="artemisApp.universityInfo.adminName"></div>
+            <div class="inline-block">{{ universityAdminName }}</div>
+        </div>
+        <div>
+            <div class="inline-block fixed-width" jhiTranslate="artemisApp.universityInfo.adminEmail"></div>
+            <div class="inline-block">{{ universityContactEmail }}</div>
```

Comment:
Good catch, I added conditions, to only show the information if it is actually available.
![image](https://github.com/user-attachments/assets/2cb52c9e-4ac4-4c38-94b1-cfecdb5b33a6)
![image](https://github.com/user-attachments/assets/e6b9193b-8291-4674-b7bf-61f817af3573)

Would it be desirable in your opinion, to make it more customizable, which information is sent to the telemetry service? E.g. allow sending telemetry, but only the university name and version, and not the contact email. ?

Reply:
Yes, I would like to always have a working ‘Contact Us‘ email link on the ‘About Us’ page.
As long as the email/name are sent via the telemetry service, though, I will definitely disable this service.

---

Note: The Readme lists at least one Hochschule and one School as Artemis operators. https://github.com/ls1intum/Artemis?tab=readme-ov-file#universities--schools-with-artemis-in-use

I think the config value names and the displayed text should be updated to be more generic (eg. ‘Betreiber’/‘Operator’ instead of ‘University’).